libr/util/table.c

```diff
@@ -70,6 +71,8 @@ R_API RTable *r_table_new() {
 		t->showHeader = true;
 		t->cols = r_list_newf (r_table_column_free);
 		t->rows = r_list_newf (r_table_row_free);
+		t->useUtf8 = true;
```

Comment:
dont enable by defafult. this should depend on user settings. maybe generalized via r_core_table_new (RCore *core)

Reply:
I cannot bring in the RCons directly here. It is giving me an error.

Reply:
Done